public/app/features/apiserver/client.ts

```diff
@@ -34,6 +38,43 @@ export class ScopedResourceClient<T = object, S = object, K = string> implements
     return getBackendSrv().get<Resource<T, S, K>>(`${this.url}/${name}`);
   }
 
+  public watch(opts?: WatchOptions): Observable<ResourceEvent<T, S, K>> {
+    const finalOpts = opts || {};
```

Comment:
I guess we'll only use this for k8s watch endpoints, so should this have `watch: true` at all times?